client/ayon_hiero/api/lib.py

```diff
@@ -1283,13 +1290,13 @@ def sync_clip_name_to_data_asset(track_items_list):
             continue
 
         # fix data if wrong name
-        if data["asset"] != ti_name:
-            data["asset"] = ti_name
+        if data["asset_name"] != ti_name:
+            data["asset_name"] = ti_name
```

Comment:
What is "data asset"? if this refers to "asset" from OpenPype/AYON pipeline perspective then this would be old terminology and should be `folder` `folder_name` or `folder_path` nowadays? Or is this a Hiero thing?

Reply:
Also, this `asset_name` key now seems to be the OPPOSITE of what we're imprinting here with this PR: https://github.com/ynput/ayon-hiero/pull/14/files#diff-7009804c53f2d17be55333b2cb25bc8a7bc63a92cee08aa8bce0ed37648e5726R748

Reply:
This seems to be addressed